fv3core/fv3core/stencils/del2cubed.py

```diff
@@ -114,7 +114,7 @@ def __init__(
             direction="x", stencil_factory=stencil_factory
         )
 
-        self._ntimes = min(3, nmax)
+        self._ntimes = int(min(3, nmax))
```

Comment:
No, these variables in serialbox are stored as float64. They have to be cast somewhere to int. I decided for init variables to do the casting in the class for type safety, since we don't care about init performance.